src/bp/migrations/v12_17_2-1612973566-add_botid_to_dialog_sessions_id_and_remove_botid_column.ts

```diff
@@ -28,10 +28,8 @@ const migration: Migration = {
 
```

Comment:
I understand your change, but I am against editing previous migrations this way. This can cause undesirable consequences and your next migration should handle it correctly

Reply:
What do you suggest then ? removing the previous migration all along ?

Reply:
As discussed with Eff, we decided it would be better to simply remove this migration.